src/librustc_metadata/creader.rs

```diff
@@ -610,6 +611,15 @@ impl<'a> CrateLoader<'a> {
                 );
                 self.0.push((Symbol::intern(trait_name), Rc::new(derive)));
             }
+
+            fn register_custom_attribute(&mut self,
```

Comment:
Could you also change these other uses of `custom_attribute` to `attr_proc_macro` (or something else if you want)?